src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceCodeAction.cs

```diff
@@ -80,7 +83,7 @@ private static ImmutableArray<CodeActionOperation> CreateRenameOperations(MoveTo
         public static AbstractMoveToNamespaceCodeAction Generate(IMoveToNamespaceService changeNamespaceService, MoveToNamespaceAnalysisResult analysisResult)
             => analysisResult.Container switch
             {
-                MoveToNamespaceAnalysisResult.ContainerType.NamedType => (AbstractMoveToNamespaceCodeAction)new MoveTypeToNamespaceCodeAction(changeNamespaceService, analysisResult),
```

Comment:
ermagerd.  so happy!  we should fix this up everywhere.   Does the 'remove unnecessary cast' fix find this?

Reply:
yup

Reply:
I'm sad I didn't catch this when I wrote it :(